app_dart/README.md

```diff
@@ -111,6 +133,22 @@ $ docker images|grep local|tr -s ' '|cut -d' ' -f3|xargs docker rmi -f
 
 ### Deploying a release to App Engine
 
+#### Auto-deploy
+Cocoon auto deployment has been set up via
+[Google Cloud Build](https://pantheon.corp.google.com/cloud-build/triggers?project=flutter-dashboard)
+daily on Workdays. Please view [details](https://g3doc.corp.google.com/company/teams/flutter/cocoon/cloud_deploy.md?cl=head)
```

Comment:
edge case: This is probably fine to help with visibility, but I think having something like a go link would be preferable

Reply:
Used go link instead.